### ISSUE TITLE
Run coveralls only in merge commit

### ISSUE THREAD

**Issue by item4** [clicheio/cliche#115]
In PR, No one can access secure environment variables. So some tests will be passed and coverage must be decreased.

I think it is insain. :(

I want to fix to run coveralls only in merge commit.

**Comment by miaekim** [comment]
I don't agree with that. what if we could not check running? we need to check our pr before merged.
